app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -132,15 +132,20 @@ class SupplementalApplicationPage extends React.Component {
   handleStatusModalSubmit = async (submittedValues) => {
     this.setState({loading: true})
     this.updateStatusModal({loading: true})
+    console.log('submittedValues', submittedValues)
 
     const data = {
       status: this.state.statusModal.status,
       comment: submittedValues.comment,
       applicationId: this.state.persistedApplication.id
     }
+    const appResponse = await updateApplicationAction(this.state.persistedApplication)
```

Comment:
@callaghanc @fddayan I thought this would be super easy, but it looks like the handleStatusModalSubmit doesn't have access to the formApi values for the application. Any ideas for what to do about it?

Reply:
This is kind of the problem I faced with the ux behavior PR and the nature form react-form. I'll ping you on slack and we can pair.